portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -73,11 +73,21 @@ function createCommentElement(comment) {
 	deleteEl.setAttribute("onclick", `deleteCommentFromStorageById(${comment.id})`);
 	deleteEl.setAttribute("src", "images/icons/trash.svg");
 
-	const authorDateDeleteContainer = document.createElement("div");
-	authorDateDeleteContainer.appendChild(authorEl);
-	authorDateDeleteContainer.appendChild(deleteEl);
-	authorDateDeleteContainer.appendChild(dateEl);
-	children.push(authorDateDeleteContainer);
+	const scoreEl = document.createElement("span");
```

Comment:
In retrospect, especially because this addition is a bit more involved, I'm thinking it might be cleaner to extract out all of these element creations into their own helper methods to keep this one more readable

Reply:
I agree, that's a good point as well. Will do that.